### PR TITLE
Satya Add Mac Explorer Application 

### PR DESCRIPTION
**Pull request by sgugulothu**
Mac Explorer Application

### CODE REVIEW

File: src/test/java/AnalysisTools/AnalysisToolsRunners/RunMCAExplorerSmokeTest.java
Comment (matakevin) [inline src/test/java/AnalysisTools/AnalysisToolsRunners/RunMCAExplorerSmokeTest.java:1]: remove empty spaces and format the tags.  For tags, you should only used the "@Smoke" tag for a smoke runner or "@Regression" for a regression runner.

File: src/test/java/AnalysisTools/MCAExplorer/Steps/MCAExplorer_pairwiseplots_steps.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
+package AnalysisTools.MCAExplorer.Steps;
+
+import org.testng.Assert;
+import org.testng.asserts.Assertion;
+
+import com.nci.automation.web.CommonUtils;
+import com.nci.automation.web.EnvUtils;
+import com.nci.automation.web.JavascriptUtils;
+import com.nci.automation.web.WebDriverUtils;
+
```
Comment (hamidnesar) [inline src/test/java/AnalysisTools/MCAExplorer/Steps/MCAExplorer_pairwiseplots_steps.java:10]: Please remove the empty line

File: src/test/java/AnalysisTools/MCAExplorer/Features/MCAExplorer.feature
Comment (matakevin) [inline src/test/java/AnalysisTools/MCAExplorer/Features/MCAExplorer.feature:1]: Smoke, Progression or Regression tags should be present.

**Review by hamidnesar — COMMENTED** [review]
Please refer the comments on each page. Thank you!

File: src/test/java/AnalysisTools/MCAExplorer/Pages/MCAExplorerPage.java
Comment (matakevin) [inline src/test/java/AnalysisTools/MCAExplorer/Pages/MCAExplorerPage.java:1]: Remove lines 15, 17, 75, 118, 145, 152, 162, 173, 186-192,

Comment (matakevin) [reply src/test/java/AnalysisTools/MCAExplorer/Pages/MCAExplorerPage.java:1]: Also, when naming WebElements we use camel casing where the first letter is lowercase.  We also want to refrain from using underscores with elements

File: src/test/java/AnalysisTools/MCAExplorer/Pages/package-info.java
Comment (matakevin) [inline src/test/java/AnalysisTools/MCAExplorer/Pages/package-info.java:1]: delete package-info subclass

File: src/test/java/AnalysisTools/MCAExplorer/Steps/HooksSteps.java
Comment (matakevin) [inline src/test/java/AnalysisTools/MCAExplorer/Steps/HooksSteps.java:1]: removing extra blank lines

File: src/test/java/AnalysisTools/MCAExplorer/Steps/MCAExplorer.java
Comment (matakevin) [inline src/test/java/AnalysisTools/MCAExplorer/Steps/MCAExplorer.java:1]: remove empty lines in the imports and lines where not necessary.  Remove any commented out code.  Its fine to attempt to comment things out on our local but there shouldn't be any commented code on the master branch

File: src/test/java/AnalysisTools/MCAExplorer/Steps/MCAExplorer_pairwiseplots_steps.java
Comment (matakevin) [inline src/test/java/AnalysisTools/MCAExplorer/Steps/MCAExplorer_pairwiseplots_steps.java:1]: remove extra blank lines and commented code

File: src/test/java/AnalysisTools/MCAExplorer/StepsImplementations/MCAExplorerStepImp.java
Comment (matakevin) [inline src/test/java/AnalysisTools/MCAExplorer/StepsImplementations/MCAExplorerStepImp.java:1]: remove extra blank lines

File: src/test/java/appsCommon/PageInitializers/PageInitializer.java
Comment (matakevin) [inline src/test/java/appsCommon/PageInitializers/PageInitializer.java:1]: remove extra blank lines

**Review by matakevin — CHANGES_REQUESTED** [review]
Review changes as requested.

**Pull request merged**